modules/apps/transfer/types/expected_keepers.go

```diff
@@ -25,6 +26,8 @@ type BankKeeper interface {
 	SendCoinsFromAccountToModule(ctx sdk.Context, senderAddr sdk.AccAddress, recipientModule string, amt sdk.Coins) error
 	BlockedAddr(addr sdk.AccAddress) bool
 	IsSendEnabledCoin(ctx sdk.Context, coin sdk.Coin) bool
+	HasDenomMetaData(ctx sdk.Context, denom string) bool
+	SetDenomMetaData(ctx sdk.Context, denomMetaData banktypes.Metadata)
```

Comment:
I think it would good to change all instances of `MetaData` to `Metadata`

Reply:
I changed the ones decalred within IBC. But didn't change the methods from bankkeeper.